__tests__/app.test.js

```diff
@@ -76,3 +97,73 @@ describe('GET/api/articles/:article_id', () => {
   });
 });
 
+describe('GET/api/articles', () => {
+  it("gets all articles", () => {
+    return request(app).get("/api/articles").then((response) => {
+            expect(response.body.articles.length).toBe(13);
+         });
+     });
+  it('GET:200 and should respond with an articles array all article objects', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        expect(Array.isArray(articles)).toBe(true)
+        articles.forEach((article) => {
+          expect(article).toHaveProperty('author');
```

Comment:
do we need to check if all these properties exist if on the following line you're accessing them using dot notation?